### PR TITLE
moderncv:0.1.0

### PR DESCRIPTION
**Pull request by giovanniberti**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: A typst template inspired by LaTeX **moderncv**

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by giovanniberti** [comment]
The test action is failing with ```failed to process package at packages/preview/moderncv/0.1.0: failed to process thumbnail: thumbnail must be a PNG or WebP image```, but on the terminal I get
```sh
$ file preview.png
preview.png: PNG image data, 1027 x 783, 8-bit/color RGBA, non-interlaced
```

Any help?

**Comment by Enivex** [comment]
There's already a template with the same name (with a hyphen)

https://typst.app/universe/package/modern-cv

**Comment by giovanniberti** [comment]
Is `moderncv.typst` ok as a package name?

**Comment by Enivex** [comment]
> Is `moderncv.typst` ok as a package name?

That's the same name, just with typst added, which is explicitly against the rules.

(The existing `modern-cv` is honestly a bit too generic in my opion, but what is done is done I guess)

**Comment by laurmaedje** [comment]
> The existing modern-cv is honestly a bit too generic in my opion, but what is done is done I guess

I understand your position and also felt that it might be, but it is, in principle, exactly named according to our rules (adjective + descriptive part).

File: packages/preview/moderncv/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+keywords = ["CV", "Curriculum Vitae", "Resume", "Minimalistic", "Professional", "Modern"]
+categories = ["cv"]
+exclude = ["example*", "moderncv.pdf"]
+homepage = "https://github.com/giovanniberti/moderncv.typst"
+repository = "https://github.com/giovanniberti/moderncv.typst"
+
+[template]
+path = "template"
+entrypoint = "main.typ"
+thumbnail = "preview.npg"
```
Comment (laurmaedje) [inline packages/preview/moderncv/0.1.0/typst.toml:18]: The thumbnail error is because of this:
```suggestion
thumbnail = "preview.png"
```

**Review by laurmaedje — COMMENTED** [review]
As @Enivex said, `moderncv.typst` doesn't work either, both because `typst` shouldn't be used in package names and because dots aren't valid in package names. I would recommend just using some completely different name, not including the word "modern". See the [submission guidelines](https://github.com/typst/packages?tab=readme-ov-file#submission-guidelines) for details. You can also take a look [at the existing templates](https://typst.app/universe/search?kind=templates) for inspiration.

**Comment by giovanniberti** [comment]
Ok, I'll guess I'll have to invent a new name, for now I'll close the PR

**Pull request closed without merging**